backend/grant/admin/views.py

```diff
@@ -587,8 +565,8 @@ def create_contribution(proposal_id, user_id, status, amount, tx_id):
     db.session.add(contribution)
     db.session.flush()
 
+    #TODO: should this stay? 
```

Comment:
Good question. I think we can just have proposals go live, the intermediate step where users can then publish their accepted proposal doesn't make sense anymore.

Reply:
Opened issue #29 for this so we don't have to add more to this PR.